### DISCUSSION TITLE
Screw contacts to RJ11 adapter board?

### DISCUSSION THREAD

**Discussion by nstadigs** [nliaudat/esp32_8ch_motor_shield#60] [Ideas]
Many in the community seem to want the RJ11 jacks for the hmip-actuators, including me. But at the same time I understand that @nliaudat wants a more universal board that can be used for many different things. And also he doesn't want to maintain multiple versions which I totally understand. And I understand that you can cut the wires but I'd rather not, personally.

How about making an adapter board with RJ11 jacks that you screw into the terminals? This could be done by the community.

Alternatively replace the screw connectors on the main board with dupont connectors and have two adapter boards. One with screw terminals and one with RJ11. That would probably require more work tho.

**Comment by nliaudat** [comment]
Perhaps things like that is the best way to go (low price) : 

![image](https://github.com/user-attachments/assets/634da90b-0deb-4119-a544-4cbb5bc551a8)


https://fr.aliexpress.com/item/1005005810724069.html

**Comment by nliaudat** [comment]
The new board design is smaller due the opamp. 
The old RJ11 do not fit

![image](https://github.com/user-attachments/assets/14ea74da-e9c6-42e9-98ad-3f3e26e0e539)
